static/app/views/performance/browser/resources/resourceSummaryPage/index.tsx

```diff
@@ -97,7 +97,11 @@ function ResourceSummary() {
           </HeaderContainer>
           <ResourceSummaryCharts groupId={groupId} />
           <ResourceSummaryTable />
-          <SampleList groupId={groupId} transactionName={transaction as string} />
+          <SampleList
+            isBrowser
```

Comment:
l: rather than making the `SampleList` have a boolean attribute, what do you think about passing in the path? So here we would do `performance/browser/pageloads`, while the default would be `/performance/summary/`.

This way `SampleList` doesn't need to be aware of routing at all.

Reply:
I'm okay with that but can but do you mind explaining the benefits of this further? 
If we passed in the route every browser module going forwards (for example interactions) will have to specify the route  everytime instead of a boolean condition which can be prone to typos.
I can see a future where the transaction name links to the page load module if browser, web service view if backend, lambda view if serverless, etc

Reply:
I'd rather the `SampleList` not be one who needs to keep track of all these routes - it should be the parent that tells the `SampleList` where to go to.

We also remove invariant conditions - there's only one path to use (what happens if `isBrowser` and `isServerless` is both true, which link should be rendered?).

> specify the route everytime

This is easy to unit test, and very obvious when it breaks, I wouldn't worry about it too much.

Reply:
👍 Thanks for the explanation! I made the change, I wasn't sure what a good variable name was here, called it `transactionRoute`.